static/css/new-case-studies.css

```diff
@@ -144,6 +151,18 @@ h4[id]:before {
   letter-spacing: 0.03em;
 }
 
+.youtube-quote-sm {
+  display: inline-block;
+  width: 380px;
```

Comment:
Hello @habibrosyad .
The shortcode changes look okay. Where is the `youtube-quote-sm` style used?

Reply:
@kbhawkey it is used for the `youtube` shortcode in the case study, e.g

```
{{< youtube DqS_IPw-c6o youtube-quote-sm >}}
```

The shortcode only supports adding css class and no other customizations such as width or height are possible at the moment.

I'm thinking of making the style more reusable for other (future) case studies. If you need small youtube frame you could use that css class (hence the `-sm` suffix, borrowing bootstrap's naming style). If you need a full width frame then you could use, let's say, just the `youtube-quote` class (haven't defined this yet)

Reply:
Thanks! I wanted to understand why you passed in the class to the shortcode.